.github/workflows/scripts/native-image-tests.sh

```diff
@@ -21,7 +21,14 @@ run_sample_tests () {
     fi
   done
   project_names="$(echo "${module_samples[@]}" | sed 's/ /,/g')"
-  mvn clean --activate-profiles native-sample-config,nativeTest --define notAllModules=true --define maven.javadoc.skip=true -pl="${project_names}" test
+  if [[ $project_names =~ "spring-cloud-gcp-pubsub-stream-functional-sample" ]]; then
+    filtered_project_names=$(echo "$project_names" | sed 's/,spring-cloud-gcp-samples\/spring-cloud-gcp-pubsub-stream-functional-sample//')
+    pushd spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-functional-sample/spring-cloud-gcp-pubsub-stream-functional-sample-test
+    mvn package -Pnative -Pnative-sample-config -DskipTests
+    mvn -Pnative-sample-config -PnativeTest --define notAllModules=true --define maven.javadoc.skip=true test
+    popd
+    mvn clean --activate-profiles native-sample-config,nativeTest --define notAllModules=true --define maven.javadoc.skip=true -pl="${filtered_project_names}" test
+  fi
 }
```

Comment:
@mpeddada1 : It looks like this logic no longer runs sample tests unless we enter this `spring-cloud-gcp-pubsub-stream-functional-sample` if-block. Is that intentional?

Reply:
Fixed, thank you.